webapp/src/dogma/common/components/Navbar.tsx

```diff
@@ -92,7 +92,7 @@ export const Navbar = () => {
   const { colorMode, toggleColorMode } = useColorMode();
   const { user } = useAppSelector((state) => state.auth);
   const dispatch = useAppDispatch();
-  const { data: projects = [] } = useGetProjectsQuery();
+  const { data: projects = [] } = useGetProjectsQuery({ admin: user.roles.includes('LEVEL_ADMIN') });
```

Comment:
I found a bug where products are null even with the default value when I deleted all projects.
<img width="959" alt="image" src="https://user-images.githubusercontent.com/1866157/218996044-87618eff-0bff-4a14-b281-fd608f6100b3.png">
https://stackoverflow.com/questions/39522014/destructuring-falsy-and-null-with-default-parameters
The server returns 204 no content and it makes `useGetProjectsQuery` return null.